src/items/type-aliases.md

```diff
@@ -4,7 +4,7 @@
 > _TypeAlias_ :\
 > &nbsp;&nbsp; `type` [IDENTIFIER]&nbsp;[_GenericParams_]<sup>?</sup>
 >              ( `:` [_TypeParamBounds_] )<sup>?</sup>
->              [_WhereClause_]<sup>?</sup> ( `=` [_Type_] )<sup>?</sup> `;`
+>              [_WhereClause_]<sup>?</sup> ( `=` [_Type_] )<sup>?</sup> [_WhereClause_]<sup>?</sup> `;`
```

Comment:
Why include `_WhereClause_` twice? The first placement is deprecated.

Reply:
Since it is supported in both positions, it needs to be included in both places.

However, this grammar does seem incorrect to me.  It seems to allow `type Assoc<'a, T> where T: 'a where T 'a;`.  I think the second where clause needs to be inside the parentheses for the `=`.

It might be good to have a sentence somewhere on this page that discusses the deprecated form. That might be a bit subtle to express clearly, since there are different forms, such as with and without `=`, and bounds on (non-associated) type aliases are ignored (`type_alias_bounds` warning).

Reply:
Yes, it's *soft* deprecated, but it's still valid syntax. I think for now it makes sense to keep it.

Reply:
I guess not "soft" deprecated, since we warn. Added a sentence.